plugins/metrics/manaresearch.go

```diff
@@ -0,0 +1,148 @@
+package metrics
+
+import (
+	"sync"
+
+	"github.com/iotaledger/goshimmer/packages/mana"
+	"github.com/iotaledger/goshimmer/plugins/autopeering/local"
+	manaPlugin "github.com/iotaledger/goshimmer/plugins/mana"
+	"go.uber.org/atomic"
+)
+
+var (
+	accessMapBM1           mana.NodeMap
+	accessPercentileBM1    atomic.Float64
+	accessLockBM1          sync.RWMutex
+	consensusMapBM1        mana.NodeMap
+	consensusPercentileBM1 atomic.Float64
+	consensusLockBm1       sync.RWMutex
+)
+
+// AccessPercentileBM1 returns the top percentile the node belongs to in terms of access mana holders.
```

Comment:
Maybe specify in the comments that it's for access mana "using BM1".
I don't feel strongly about this though. (if you disagree)